Dockerfile.spark

```diff
@@ -0,0 +1,29 @@
+FROM gettyimages/spark:2.1.0-hadoop-2.7
+
+RUN apt-get update && apt-get install -yq --no-install-recommends --force-yes \
+    git \
+    openjdk-7-jdk \
+    maven && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV LIVY_BUILD_VERSION livy-server-0.3.0
+ENV LIVY_APP_PATH /apps/$LIVY_BUILD_VERSION
+ENV LIVY_BUILD_PATH /apps/build/livy
```

Comment:
I've made the other changes, but I'm not sure I understand the need for this one. The base image I'm using for the Livy container (`gettyimages/spark:2.1.0-hadoop-2.7`) already has a Python3 installation which seems to be working just fine with Livy and sparkmagic as-is.

Do you just mean creating a virtualenv for it? I can do that, though I'm not sure it's necessary if we're just running inside a single-purpose container.